src/test/com/softserve/temproject/controller/LoginControllerTest.java

```diff
@@ -0,0 +1,38 @@
+package com.softserve.temproject.controller;
+
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+import com.softserve.teamproject.controller.LoginController;
+import java.io.IOException;
+import javax.servlet.http.HttpServletResponse;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+public class LoginControllerTest {
+
+  @Mock
+  HttpServletResponse response;
+
+
+  @Before
+  public void init() {
+    MockitoAnnotations.initMocks(this);
+  }
+
+
+  @Test
+  public void notAllowedMethod_GetRequest_NotAllowedReturned() throws IOException {
```

Comment:
name of method should be in CamelCase so no underscores, also try to make it shorter. notAllowedMethodTest would be ok. Also could you fix this problem in UserDetailsServiceImplTest.java too